CI/circle_parallel.sh

```diff
@@ -33,6 +33,12 @@ elif [ "$NODE_INDEX" = "2" ]; then
   if [[ $project_version == *"-SNAPSHOT" ]]; then
     echo "Running node $NODE_INDEX to test ensure-up-to-date"
     java -version
+    # Install golang version 1.14
+    go version
+    sudo mkdir /usr/local/go1.14
+    wget -c https://dl.google.com/go/go1.14.linux-amd64.tar.gz -O - | sudo tar -xz -C /usr/local/go1.14
```

Comment:
Why is all this necessary?

Reply:
it's probably to run checks against latest version of go, but this looks far out of scope for this projects and IMO shouldn't be here, that's another thing to remember and maintain.